packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -30,10 +29,8 @@ export class AtomicResultPrintableUri {
   @ResultContext() private result!: Result;
 
   @State() listExpanded = false;
-  @BindStateToI18n()
-  @State()
```

Comment:
Why does switching to kebab case involve removing these two lines?

Reply:
Since Im touching the same files, I also removed this deprecated method